packages/peregrine/lib/talons/CartPage/GiftOptions/useGiftOptions.js

```diff
@@ -0,0 +1,130 @@
+import { useState, useCallback, useEffect, useMemo } from 'react';
+import throttle from 'lodash.throttle';
+import { useMutation, useLazyQuery } from '@apollo/react-hooks';
+
+import { useCartContext } from '@magento/peregrine/lib/context/cart';
+
+const useGiftOptions = ({ getGiftOptionsQuery, saveGiftOptionsQuery }) => {
```

Comment:
Avoid destructuring here. The pattern in our codebase is to just call this object `props`, and destructure it in the function body.